### PR TITLE
Fixing twitter scraping to reflect new displaying requirements

### PR DESCRIPTION
**Pull request by caolan-power**
No issue to attach to this as it's a bit of an impromptu one. 

Need to change the lambda function to scrape extra data from twitter now that I am making the tweet component from scratch. Not a huge PR

### CODE REVIEW

**Comment by caolan-power** [comment]
Forgot to turn formatter off editing files that weren't mine. Only real changes are to tweet model, tweet controller and index.py in twitter scraping lambda. sorry

File: amplify/backend/function/expressServer/src/controllers/stockController.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const getStockPrice = async (req, res, next) => {
   }
 };
 
-// @route   GET api/stocks
+// @route   GET api/stock
 // @desc    Get all stocks
 // @access  Private - add auth middleware to make it private
 const getAllStocks = async (req, res, next) => {
```
Comment (BearachB) [inline amplify/backend/function/expressServer/src/controllers/stockController.js:30]: @caolan-power Is this getAllStocks function supposed to be here?

Comment (caolan-power) [reply amplify/backend/function/expressServer/src/controllers/stockController.js:30]: Nope and can't see it on my local machine so strange

Comment (caolan-power) [reply amplify/backend/function/expressServer/src/controllers/stockController.js:30]: should be ok now

**Review by BearachB — APPROVED** [review]
Looks good overall, recreating the Tweets will give us a lot more flexibility in what we display!

**Pull request merged**